client/stylesheets/_toCat.scss

```diff
@@ -4753,6 +4753,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .searchResultsCardRight li b {
     margin-right: 3px;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces